src/screencast/screencast.c

```diff
@@ -419,7 +419,7 @@ static int method_screencast_start(sd_bus_message *msg, void *data,
 		start_screencast(cast);
 	}
 
-	while (cast->node_id == 0) {
+	while (cast->node_id == SPA_ID_INVALID) {
```

Comment:
I think this was working fine before because `cast` gets zero-initialized. But `SPA_ID_INVALID != 0`, so we now need to explicitly initialize `node_id` when we create the stream. Or else we'll never enter this loop and end up with a zero node ID.

Can youa dd this in e.g. `xdpw_pwr_stream_create`?

Reply:
Added it to `xdpw_screencast_instance_init` since the other initializations take place there.